build.go

```diff
@@ -19,91 +17,55 @@ type DependencyManager interface {
 	GenerateBillOfMaterials(dependencies ...postal.Dependency) []packit.BOMEntry
```

Comment:
We should be able to remove this method with these changes

Reply:
Yes, you're right! There are a few other changes I'd like to make before proposing this for merge. I'd like to wait for the underlying packit library interface to stabilize, and then clean this PR up.

Can you add the `draft` label to this PR? I can't do that myself.